adapter/validator/src/test/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidatorTest.java

```diff
@@ -38,11 +38,11 @@
     public static final String TEST_FILE_TST = "test_file.tst";
 
     @Test
-    void inRangeFloatDoNotGenerateNotice() {
+    void simpleFloatDoNotGenerateNotice() {
```

Comment:
the meaning of simple in that context is unclear. suggestions: regular, typical, random